core/src/main/java/org/n52/iceland/ogc/swes/SwesExtension.java

```diff
@@ -25,7 +25,6 @@
  *
  * @since 1.0.0
  */
-@Deprecated // SOS-specific and not constants
```

Comment:
That's actually not SOS but SWES... It's not pure OWS but definitely not service specific... Maybe we should split `iceland-swes` in some later release...